_posts/2016-08-17-improve-teams-communication-with-user-stories.md

```diff
@@ -0,0 +1,88 @@
+
+**The Confirmation** is the criteria for when a story can be considered "DONE". The criteria may be written as acceptance tests.
+
+A User Story is always written from the perspective of the user. Common description formats that can help teams new to Agile practices as SCRUM are:
+
+> As a < user role > I < want, need, can, etc > < goal > so that < reason >
+
+> In order to < reason > as a < user role > I < want, need, can, etc > < goal > 
+
+The formats help keep the focus on the right questions to ask to reduce the risk of getting into technical details. 
+
+### Who writes User Stories?
+
+Everyone is involved in the creation of user stories: customers, users and developers. The stories go into the Product Backlog and the Product Owner has the responsibility of deciding which one to keep and which one won’t be implemented.
+
+### What makes a good User Story?
+
+The elements of a good User Story can be summarised with the acronym INVEST:
+
+ - Independent
+ - Negotiable
+ - Valuable to users and customers
+ - Estimatable
+ - Small
+ - Testable
+
+User Stories should be **INDEPENDENT** of one another to avoid prioritisation and planning issues. An independent story can be implemented at any time of the or removed from the product backlog without having a negative impact on the other stories. 
+
+Stories are not requirements written in stones; stories are **NEGOTIABLE**. During the conversation phase, the Customer Team and the Development Team discuss the details and requirements needed to implement the story. 
+
+A Story must be **VALUABLE** to the user (who will use the product) or the customer (who will buy the product). This is the most important part of a User Story. The best way to achieve this is to have both real users and customers in the Customer Team so that they can write stories by themselves.
+
+A Story must be **ESTIMATABLE**. Developers must be able to understand how much time it will take to complete a story. There are few reasons why sometimes it is not possible to estimate a story:
+
+ - the goal and the description is not clear to them
+ - they lack in technical knowledge 
+ - the story is too big
+
+A **SMALL** story is easier to understand, estimate and break into tasks. A big story that can be broken into multiple shorter stories is called Epic.
+
+A story is **TESTABLE**. Only if it succeeds in passing acceptance tests a story can be considered complete and the customer can have a new piece of working software.
+
+
+### Conclusion
+
+User Stories have been designed to build shared understanding between the Customer Team and the Developer Team. They are conversations that lead an agreement on what to build and their format helps teams to keep the focus on delivering features valuable to business and users. That’s why they fit really well in an iterative and incremental development process.
```

Comment:
What is also worth mentioning about User Stories is that they help communicate across various stakeholders and participants of a project via a ubiquitous (common) language.